### ISSUE TITLE
Feature Request: edit environment

### ISSUE THREAD

**Issue by chantisnake** [James-Yu/LaTeX-Workshop#612]
Is it possible to provide a command to edit the environment?

For example:
```latex
\begin{theorem}
  <cursor here>
\end{theorem}
```

Press `ctrl+shift+P`, select `LaTeX-WorkShop: edit environment`
and then enter `lemma`, will produce:
```latex
\begin{lemma}
  <cursor here>
\end{lemma}
```

**Comment by jlelong** [comment]
You can use the  _LaTeX Workshop: Multicursor selection of the current environment name_ function (see the last item of the Features section in the [README](https://github.com/James-Yu/LaTeX-Workshop#features)). It creates a multi-cursor positioned on the first letter of the environment name.

**Comment by jlelong** [comment]
You screenshot looks strange: no begin in front of `{theorem}`.

From your example, running _LaTeX Workshop: Multicursor selection of the current environment name_ yields

<img width="155" alt="capture d ecran 2018-05-08 a 18 50 35" src="https://user-images.githubusercontent.com/2910140/39770923-c4b2d6a2-52f0-11e8-8f8b-5ca6c8bda187.png">

The two red boxes are a multi-cursor.

**Comment by chantisnake** [comment]
Uh, That is just the result of [prettify symbol](https://github.com/siegebell/vsc-prettify-symbols-mode).

I was kind of hopping that the plugin can select the environment for me. Like this:
![capture](https://user-images.githubusercontent.com/10100995/39772124-5348e89a-52c2-11e8-81b5-b4cead252240.PNG)

**Comment by chantisnake** [comment]
If you just selected the env, you can just press `rightarrow`, and then `*` to insert the star.
I personally think this more intuitive than `ctrl+rightarrow` and then `*` (or `e` and then `i` `*` if you are using vim, I believe).

**Comment by jlelong** [comment]
> you are using vim, I believe 

You believe right and it does not make any difference to have a multi-cursor (`cw` deletes the environment) or a selection (use `c` instead)  in that case.

**Comment by chantisnake** [comment]
that makes sense if you don't want to select the environment, but for people that are not using vim, I believe selection by default would make more sense.

Maybe we can write an option to set the behavior to `select`, `cursor at end of env` or `cursor at begin of env`

**Comment by chantisnake** [comment]
I can look into it after I graduate (10 days to go!).

**Comment by jlelong** [comment]
Going back to the vim mode, the problem is that I could not find how to enter insert mode at the end of a multi selection (which does not exist natively  in Vim).

**Comment by chantisnake** [comment]
Yes. I understand. But given the name is `LaTeX Workshop: Multicursor selection of the current environment name`, user probably will expect the name to be selected.
I suggest we create 3 different commands:
- `LaTeX Workshop: Multicursor select current environment name`
- `LaTeX Workshop: Multicursor go to the start of current environment name`
- `LaTeX Workshop: Multicursor go to then end of current environment name`

**Comment by jlelong** [comment]
You are right, the function name is maybe a bit misleading.

I think, two different commands are enough:
- `LaTeX Workshop: Multicursor select current environment name`
- `LaTeX Workshop: Set multicursor on the current environment name`.

For maintenance purposes, it is always better to keep things as simple as possible...

**Comment by chantisnake** [comment]
I think `to start` may be useful to vim user like yourself, and `to end` maybe useful for regular user who just want to add a `*` at the end. 

I personally will only use the `select` feature, so as long as `select` is in the mix, I will be happy.

We should probably discuss with @James-Yu

**Comment by James-Yu** [comment]
As a non-vim user, I agree with @chantisnake that `select` sounds like a multi-selection rather than set a multi-cursor. Yet the two commands by @jlelong looks good enough.

**Comment by jlelong** [comment]
OK, I will make two functions and rename the current one to _Set multicursor_ instead of _Select_.
